### ISSUE TITLE
[fmt:x64-windows] build failure

### ISSUE THREAD

**Issue by VladBolotov** [microsoft/vcpkg#12589]
**Host Environment**
- OS: Windows 10
- Compiler:  C/C++ Optimizing Compiler Version 19.26.28806

**To Reproduce**
`vcpkg.exe install fmr:x64-windows`

**Failure logs** 
```
vcpkg.exe install fmt:x64-windows
Computing installation plan...
The following packages will be built and installed:
    fmt[core]:x64-windows
Detecting compiler hash for triplet x64-windows...
Starting package 1/1: fmt:x64-windows
Building package fmt[core]:x64-windows...
-- Using cached C:/Tools/vcpkg/downloads/fmtlib-fmt-19bd751020a1f3c3363b2eb67a039852f139a8d3.tar.gz
-- Cleaning sources at C:/Tools/vcpkg/buildtrees/fmt/src/52f139a8d3-70a26f0c4b.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Tools/vcpkg/downloads/fmtlib-fmt-19bd751020a1f3c3363b2eb67a039852f139a8d3.tar.gz
-- Applying patch fix-warning4189.patch
-- Using source at C:/Tools/vcpkg/buildtrees/fmt/src/52f139a8d3-70a26f0c4b.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: C:/Tools/vcpkg/packages/fmt_x64-windows/share/fmt/copyright
-- Fixing pkgconfig
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:279 (find_program):
  Could not find PKGCONFIG using the following names: pkg-config
Call Stack (most recent call first):
  ports/fmt/portfile.cmake:44 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:79 (include)


Error: Building package fmt:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: fmt:x64-windows
  Vcpkg version: 2020.06.15-nohash
```

**Comment by PhoebeHui** [comment]
@VladBolotov, this issue would be fixed by PR #12569, could you try the changes? sorry for blocking you.

Duplicated to #12565

**Comment by VladBolotov** [comment]
@PhoebeHui, Works for me, thanks! 

> sorry for blocking you. 

Do not worry, it's okay ;)
